public/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+		<meta name="description" content="">
+		<meta name="author" content="">
+		<link rel="icon" href="../../favicon.ico">
+
+		<title>OpenFisca - Turn law into software</title>
+
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+		<!-- Optional theme -->
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+		<!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+		<link href="../../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
+
+		<!-- Custom styles for this template -->
+		 <link href="cover.css" rel="stylesheet">
+
+
+		<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+		<!--[if lt IE 9]>
+		  <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+		  <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+		<![endif]-->
+	</head>
+
+	<body>
+		    <div class="site-wrapper">
+      <div class="site-wrapper-inner">
+        <div class="cover-container">
+          <div class="masthead clearfix">
+            <div class="inner">
+              <h3 class="masthead-brand">OpenFisca</h3>
+              <nav>
+                <ul class="nav masthead-nav">
+                  <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Country Packages<span class="caret"></span></a>
+                    <ul class="dropdown-menu">
+                      <li class="dropdown-header">France</li>
+                      <li><a href="https://github.com/openfisca/openfisca-france">GitHub repo</a></li>
+                      <li><a href="https://legislation.openfisca.fr/">Legisation explorer</a></li>
+                      <li><a href="https://legislation.openfisca.fr/swagger">Public API</a></li>
+                      <li role="separator" class="divider"></li>
+                      <li class="dropdown-header">Tunisia</li>
+                      <li><a href="https://github.com/openfisca/openfisca-tunisia">GitHub repo</a></li>
+                      <li role="separator" class="divider"></li>
+                      <li class="dropdown-header">Senegal</li>
```

Comment:
What is all this? You're taking the long road to #1, taking some of #4 along the way. This was not what we had agreed on during the planning session and means we'll have to maintain the header here.